src/HttpRequestHandler.cpp

```diff
@@ -25,6 +28,33 @@ HttpResponse HttpRequestHandler::handleRequest(HttpRequest input)
 	return HttpResponse(ExceptionManager::getErrorStatus(InternalException("Something went wrong")), "");
 }
 
+bool HttpRequestHandler::findCgi(std::string uri)
+{
+	size_t found = uri.find("/cgi_bin");
```

Comment:
I changed this to be cgi-bin so with ot-an-underscore!

Reply:
in your branch or should it be changed in the master already? In mine it was still with an underscore, but I can change it as well

Reply:
No, in mine so we can change it later / whoever goes last